client/signup/main.jsx

```diff
@@ -104,22 +101,6 @@ function dependenciesContainCartItem( dependencies ) {
 	);
 }
 
-function addLoadingScreenClassNamesToBody() {
-	if ( ! document ) {
-		return;
-	}
-
-	document.body.classList.add( 'has-loading-screen-signup' );
```

Comment:
There are existing styles for this class that we can remove.

Reply:
After several more clean-up commits, I can't find this class is in-use anymore. If I'm missing anything, please feel free to let me know :)